assets/js/ledyer-checkout-for-woocommerce.js

```diff
@@ -444,6 +443,13 @@ jQuery(function ($) {
             $(document).on('ledyerCheckoutOrderComplete', function (event) {
                 lco_wc.logToFile('ledyerCheckoutOrderComplete from Ledyer triggered');
 
+                if (lco_wc.redirectUrl !== null) {
+                    // This means that placeLedyerOrder was called successfully already
+                    // (Due to an earlier call caused by client validation)
+                    window.location.href = lco_wc.redirectUrl;
+                    return;
+                }
+
```

Comment:
See comment: https://github.com/ledyer/ledyer-checkout-for-woocommerce/pull/16/files#r1012197341